src/renderer/families/stellar/MemoTypeField.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, { useCallback, useState, useEffect } from "react";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { StellarMemoType } from "@ledgerhq/live-common/lib/families/stellar/types";
+import Select from "~/renderer/components/Select";
+import invariant from "invariant";
+import type { Account, Transaction } from "@ledgerhq/live-common/lib/types";
+
+const MemoTypeField = ({
+  onChange,
+  account,
+  transaction,
+}: {
+  onChange: string => void,
+  account: Account,
+  transaction: Transaction,
+}) => {
+  invariant(transaction.family === "stellar", "MemoTypeField: stellar family expected");
+
+  const bridge = getAccountBridge(account);
+
+  const options = StellarMemoType.map(type => ({
```

Comment:
move this in the global module scope. i mean before the `cost MemoTypeField = ({` so it's a static variable.